examples/main/package.json

```diff
@@ -13,11 +13,31 @@
     "immutable": "^3.8.1",
     "react": "^15.4.1",
     "react-dom": "^15.4.1",
-    "react-map-gl": ">=3.0.0-alpha.13"
+    "react-map-gl": ">=3.0.0-alpha.14"
   },
   "devDependencies": {
-    "buble-loader": "^0.4.0",
```

Comment:
Are we removing all examples of how to use react-map-gl with buble? Maybe keep one `exhibit`?